### PR TITLE
feat(components-native): Make react native components fonts render on Storybook

### PR DESCRIPTION
**Pull request by rodrigoeidelvein**
## Motivations

As the fonts on jobber-mobile used to use a file directly as reference to render them, it got a little bit different here as we don't want to expose the font files to everyone. Then, I came up with the following solution: Check the platform on `Typography` and render the font family in web as if they were web components.

### Added

- 

### Changed

- The way we decide the font that is going to be rendered in `Typography`

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

- run `git checkout cdb2683`
- `npm start`
- Go to the [Atlantis Typography](http://localhost:6005/?path=/docs/atlantis-text-and-typography-typography-internal--typography) page and play around with the fonts. You can compare with what [we currently have on production](https://atlantis-mobile.jobber.dev/?path=/docs/atlantis-text-and-typography-typography-internal--typography)

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components-native/src/Typography/Typography.style.ts [hunk 1]
```diff
@@ -58,6 +43,31 @@ export const typographyTokens: { [index: string]: TextStyle } = {
...
+/**
+ * `StyleSheet` for Typography.tsx.
+ *
+ * If you find yourself needing to use what's inside this object on files other
+ * than `<Typography />`, please import from `@jobber/components-native` instead.
+ *
+ * ```
+ * import { typographyStyles } from "@jobber/components-native"
+ * ```
+ */
```
Comment (darryltec) [inline packages/components-native/src/Typography/Typography.style.ts:62]: unrelated but i see it now, this comment is talking about the line 373 https://github.com/GetJobber/atlantis/blob/21624543aa08c13320e3b1e2731a9549e243f16f/packages/components-native/src/Typography/Typography.style.ts#L373

File: packages/components-native/src/Typography/Typography.style.ts [hunk 2]
```diff
@@ -58,6 +43,31 @@ export const typographyTokens: { [index: string]: TextStyle } = {
   displayRegularBlack: {
     fontFamily: "jobberpro-blk",
   },
+};
+
+const fonts = Platform.select({
```
Comment (darryltec) [inline packages/components-native/src/Typography/Typography.style.ts:48]: Maybe explain why we doing this for future us

File: packages/components-native/src/Typography/Typography.style.ts (hunk 1 above)
Comment (rodrigoeidelvein) [reply packages/components-native/src/Typography/Typography.style.ts]: Nice catch, I changed it to be above the `typographyStyles` const.

File: packages/components-native/src/Typography/Typography.style.ts (hunk 2 above)
Comment (rodrigoeidelvein) [reply packages/components-native/src/Typography/Typography.style.ts:54]: I added a comment. Not sure if it's the clearest comment, I'm accepting suggestions 😄

Comment (darryltec) [reply packages/components-native/src/Typography/Typography.style.ts:54]: It sounds good to me!

**Review by darryltec — APPROVED** [review]
LGTM :shipit:

**Pull request merged**